src/main/java/by/exadel/internship/config/jwt/JwtUtils.java

```diff
@@ -0,0 +1,47 @@
+package by.exadel.internship.config.jwt;
+
+
+import by.exadel.internship.entity.UserDetailsImpl;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.MalformedJwtException;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Component;
+
+import java.util.Date;
+
+@Component
+public class JwtUtils {
```

Comment:
From my understanding Util is something with static methods. If you are depending on @Value it more looks like service

Reply:
Fixed